src/ListenerProxy.php

```diff
@@ -67,11 +66,14 @@ public function addListener(string $methodName, int $priority = 0, string $id =
      * @return string
      *   The opaque ID of the listener.  This can be used for future reference.
      */
-    public function addListenerBefore(string $pivotId, string $methodName, string $id = null, string $type = null): string
-    {
-        $type = $type ?? $this->getParameterType([$this->serviceClass, $methodName]);
+    public function addListenerBefore(
+        string $pivotId,
+        string $methodName,
+        string $id = null,
+        string $type = null
+    ): string {
```

Comment:
Eep.  Your IDE did a bad thing. :smile: (I hate vertical parameter lists.  Especially on a trivially short method.)

Reply:
Not the IDE, the style cleaner. It was throwing two warnings, both for >120 char lines. I can set it to ignore those for the project, though.